index.d.ts

```diff
@@ -705,6 +705,12 @@ declare namespace Moleculer {
 		strategy?: Function | string;
 		strategyOptions?: GenericObject;
 		preferLocal?: boolean;
+		discoverer?: RegistryDiscovererOptions | string;
```

Comment:
It can be an instance of `BaseDiscoverer`, as well

Reply:
Yes you are right, i will make the changes. 
I tried to keep it very simple, but on second thought it is better to make it more complete as in 'Strategies';